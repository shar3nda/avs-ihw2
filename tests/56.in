\pq<yZ~;wy)(B6`X\X|Xn"o&rxuvsBPzLJ_y+:Ao=;)LJ]Q;:/d!<C6CC>0Pz*\$+#8Kucjebw& _{
Oa