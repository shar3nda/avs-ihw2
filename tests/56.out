Lowercase: 559, uppercase: 520